templates/web/brent/waste/services.html

```diff
@@ -46,9 +46,17 @@
     <dl class="govuk-summary-list">
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">Subscription</dt>
+        [%# unit.gardent_bins is a key value pair of code-number to number of bins and corresponds 1:1, 2:2 etc,
+        until it gets to 9 when it corresponds to sacks %]
+        [% IF unit.garden_bins == 9 %]
```

Comment:
Whilst this works fine, I think it'd be better to do this check in the Brent.pm cobrand code where garden_bins is read/set - if the Value is 9, set garden_sacks to 1 instead of garden_bins, perhaps, set the $garden_cost to garden_waste_sacks_cost_pa, and then here in the template you can have if garden_sacks is set and the same output.